core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -803,6 +804,22 @@ trait EntityComponent {
         entityAttributeShardQuery(workspaceContext).deleteAttributes(workspaceContext, entityType, attributesNames)
     }
 
+    def doesAttributeNameAlreadyExist(workspaceContext: Workspace,
+                                      entityType: String,
+                                      newAttributeName: AttributeName): ReadAction[Option[Boolean]] = {
```

Comment:
nit: change argument `newAttributeName` to `attributeName` - this method doesn't care if it's looking up a "new" or "old" attribute